src/js/components/generateEF/GenerateEFPage.jsx

```diff
@@ -8,6 +8,8 @@ import Navbar from '../SharedComponents/navigation/NavigationComponent.jsx';
 import AddDataHeader from './../addData/AddDataHeader.jsx';
 import Progress from '../SharedComponents/ProgressComponent.jsx';
 
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
```

Comment:
Why is this being passed in and not used? I assume you meant to add the GTAS banner to this page too? If not, this should be removed.